src/app/home/home.component.html

```diff
@@ -0,0 +1,21 @@
+<div class="homeContent">
```

Comment:
Personnaly I prefer the `Kebab case` for class name, but it's like you want.

I think it's clearer for the BEM methodology, later for the button under we will have `home-content__button`  and no `homeContent__button`.

PS: I'm open to debate :)